src/jloptions.c

```diff
@@ -130,7 +132,8 @@ static const char opts[]  =
     "                           interface if supported (Linux and Windows) or to the number of CPU\n"
     "                           threads if not supported (MacOS) or if process affinity is not\n"
     "                           configured, and sets M to 1.\n"
-    " --gcthreads=N             Use N threads for GC, set to half of the number of compute threads if unspecified.\n"
+    " --gcthreads=M[,N]         Use M threads for the mark phase of GC and N (0 or 1) threads for the concurrent sweeping phase of GC.\n"
+    "                           M is set to half of the number of compute threads and N is set to 0 if unspecified.\n"
```

Comment:
So we default to disabled?

Reply:
Yes.

Reply:
There is a tradeoff here. If you are latency bound you want to do concurrent page freeing. If you are throughput bound you won't want to oversubscribe the OS.